web/manifest/manifest.go

```diff
@@ -0,0 +1,200 @@
+package manifest
+
+import (
+	"context"
+	"fmt"
+	"html/template"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/google/go-github/v45/github"
+	"github.com/quickfeed/quickfeed/internal/env"
+)
+
+// TODO(meling) Should reuse those in env package.
+const (
+	defaultPath  = "internal/config/github/quickfeed.pem"
+	appID        = "QUICKFEED_APP_ID"
+	appKey       = "QUICKFEED_APP_KEY"
+	clientID     = "QUICKFEED_CLIENT_ID"
+	clientSecret = "QUICKFEED_CLIENT_SECRET"
+)
+
+type Manifest struct {
```

Comment:
Do we need to export this struct type and several of the methods below? Looks like it is only `StartFlow` that is used from the outside?